test/mongo_test.exs

```diff
@@ -28,6 +28,23 @@ defmodule Mongo.Test do
     end
   end
 
+#  test "show_collections", c do
```

Comment:
What was the purpose of commenting out this test?

Reply:
Ok. I forgot this test. I will fix it.